.devcontainer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Update the VARIANT arg in docker-compose.yml to pick a PHP version: 7, 7.4, 7.3
+ARG VARIANT=7
```

Comment:
Same as before, can we bump to 7.4?

Reply:
Should we follow a minor version instead of major?

Reply:
Will bump to 7.4 but we will need to confirm its use with the stakeholders

Reply:
#50